### ISSUE TITLE
Sidebar - Move course options to "Print & Export"

### ISSUE THREAD

**Issue by johnksv** [perliedman/o-scout#2]
Hi!
Just a suggestion to make the UI a bit more tidy (IMHO).

**Current:**
P.t. the sidebar UI is as following:
<img width="234" alt="Screenshot 2022-07-20 at 14 41 13" src="https://user-images.githubusercontent.com/10381866/179984481-d2061ed2-912e-40a1-be6d-54d2fc55af83.png">
As we can see, each course has an "Options"-section with settings related to print.

**Suggestion:**
Move the course options related to print to the "Print & export" section. The reasoning for this is in a workflow print & export is often the last thing you do, and the sidebar invites to follow that workflow by starting with the foldables at the "top" and "working" its way down.
E.g. my workflow would optimally be:
1. Set scale (most of the time equal to the scale of the map)
2. Add courses
3. Print/Export

I've included a quick mockup (i.e. not implemented nothing, just played around with devtools) for a suggestion of what moving the "options" to "Print & Export"  might look like.
In the mockup I've changed the `Print selection` to a drop down such that selecting the right course might be easier. I'm not sure how doable this is from a technical point, as I'm not familiar with the code. 

<img width="233" alt="Screenshot 2022-07-20 at 14 40 45" src="https://user-images.githubusercontent.com/10381866/179984524-dba361e3-45d0-4f24-95bb-454f840063ef.png">

**Comment by perliedman** [comment]
I agree on the workflow, and that's sort of how I thought about the UI. Moving the print area settings to print makes a lot of sense, _but_ there is one issue: the print options can be set per course.

I think in most cases you'd use the same area/options for all courses, but also want to have the possibility to change that. So what I want to avoid is to have another list of all courses under the print & export section.

So, I'd say I'm positive to your suggestion, just have to think a bit on how to do it without making it too complicated to handle different settings per course.
